parquet2hive/parquet2hive

```diff
@@ -100,6 +102,11 @@ def avro2sql(avro, name, version, location, partitions, with_version=True):
     else:
         partition_decl = ""
 
+    # check for duplicated fields
+    field_names = [field["name"] for field in avro["fields"]]
+    duplicate_columns = set(field_names) & set(partitions)
+    assert not duplicate_columns, "Columns {} are in both the table columns and the partitioning columns; they should only be in one or another".format(", ".join(duplicate_columns))
```

Comment:
Basically, it would say something along the lines of "partitioning column duplicated", but not which columns, or what the correct version would look like. This message simply makes sure users know exactly how to make things right.